### ISSUE TITLE
Double check user password for stronghold backup during setup

### ISSUE THREAD

**Issue by HBMY289** [iotaledger/firefly#318]
## Description

When creating a new profile and backing the seed up as a stronghold file the user you are asked to enter a password for the file once. 
After that you can start using the account.

## Motivation
 I am sure many users will not get the severity of entering/remembering the correct password here. If you put in a typo in the password for example you will never notice until you eventually need the backup and it's too late.

## Requirements

After choosing the stronghold file as backup option you should be asked to import the file again. You will have to enter the correct password to enable the import. Only after that the account should be available.
It might also be a good idea to add a text that suggests keeping this file at a safe location that is probably not the same computer.

## Are you planning to do it yourself in a pull request?

No.

**Comment by Frodooio** [comment]
Isn't the stronghold backup phrase the same as the profile password? It won't continue if you don't type your profile password, so no need to type it twice. (That was already done before).

**Comment by cvarley100** [comment]
There's no explicit need for this. We have a backup in the forced storing of the recovery phrase. If they forget their password they can return to the written backup.
